src/api/namespaces/atomicmarket/format.ts

```diff
@@ -60,27 +60,24 @@ export function formatBuyoffer(row: any): any {
 }
 
 export function formatSale(row: any): any {
-    const data = {...row};
+    const {raw_price, sale_state, offer_state, ...data} = row;
```

Comment:
CHanging the value of the attributes of a function is a little bit hard to test, wont be better to duplicate the object and then changing it and return it without modifying the other value?

Reply:
This doesn't change the passed in object - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment